components/scream/src/physics/p3/p3_functions_ice_cldliq_wet_growth_impl.hpp

```diff
@@ -0,0 +1,79 @@
+#ifndef P3_FUNCTIONS_ICE_CLDLIQ_WET_GROWTH_IMPL_HPP
+#define P3_FUNCTIONS_ICE_CLDLIQ_WET_GROWTH_IMPL_HPP
+
+#include "p3_functions.hpp" // for ETI only but harmless for GPU
+#include "p3_functions_math_impl.hpp"
+
+namespace scream {
+namespace p3 {
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>
+::ice_cldliq_wet_growth(const Spack& rho, const Spack& temp, const Spack& pres, const Spack& rhofaci, const Spack& f1pr05,
+                        const Spack& f1pr14, const Spack& xxlv, const Spack& xlf, const Spack& dv,
+                        const Spack& kap, const Spack& mu, const Spack& sc, const Spack& qv, const Spack& qc_incld, 
+                        const Spack& qitot_incld, const Spack& nitot_incld, const Spack& qr_incld, 
+                        Smask& log_wetgrowth, Spack& qrcol, Spack& qccol, Spack& qwgrth, Spack& nrshdr, Spack& qcshd)
+{
+   constexpr Scalar qsmall = C::QSMALL;
+   constexpr Scalar tmelt  = C::Tmelt;
+   constexpr Scalar twopi  = C::Pi*sp(2.0);
```

Comment:
Please use integer literals where possible, so you don't need to worry about overpromotion.
```
constexpr Scalar twopi  = C::Pi*2;
```